src/js/components/SidePanel/sidePanel.style.js

```diff
@@ -29,12 +29,20 @@ export const SidePanelWrapper = styled.div`
     transition: left .1s, width .1s;
 
     @media (min-width: ${PAGE_BREAK}px) {
-        width: ${({ renderedGraph }) => (renderedGraph === null ? '100%' : '40%')}
+        position: ${({ expanded, renderedGraph }) => {
+    if (!expanded || !renderedGraph) return 'fixed';
+    return 'relative';
+  }};
+        margin-left: ${({ expanded, renderedGraph }) => {
+    if (!expanded && !renderedGraph) return '-100%';
+    if (!expanded && renderedGraph) return '-35%';
+    return '0';
+  }};
+        width: ${({ renderedGraph }) => (renderedGraph === null ? '100%' : '35%')}
```

Comment:
Why is it so oddly formatted, is it ok?

Reply:
yeach that look horrible in my opinion, but my friend eslint told me this is only way.